product_pricelist_supplierinfo/README.rst

```diff
@@ -17,23 +17,24 @@ Supplier info prices in sales pricelists
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
-    :target: https://github.com/OCA/product-attribute/tree/16.0/product_pricelist_supplierinfo
+    :target: https://github.com/OCA/product-attribute/tree/17.0/product_pricelist_supplierinfo
```

Comment:
This is in the migration commit, but it should be on a separate commit that includes precommit changes. Presentation should be:
- Module history from the previous version (several commits
- A commit running precommit, named `[IMP] module_name: pre-commit auto fixes`
- The actual migration commit. Here you should remove `[17.0]`, this should only be included in PR's title

Reply:
@luisg123v thanks for the review. I'll correct and re-push when I get some time.